media_remover/src/main/scala/MediaNotRequiredMessageProcessor.scala

```diff
@@ -424,29 +409,34 @@ class MediaNotRequiredMessageProcessor(asLookup: AssetFolderLookup)(
       } yield maybeMd5
   }
 
-  def NOT_IMPL_getChecksumForOnline(vsItemId: String): Future[Option[String]] = {
-    // TODO? How to get the -- md5 -- checksum for a vidispine item? VS defaults to SHA-1 unless overridden in configuration property 'fileHashAlgorithm'
-    ???
+
+  def STUB_ALWAYS_RETURNS_NONE_getChecksumForOnline(vsItemId: String): Future[Option[String]] = {
+    // How to get the -- md5 -- checksum for a vidispine item? VS defaults to SHA-1 unless overridden in configuration property 'fileHashAlgorithm'
+    // FIXME Read the value of FileDocument/metadata/field/key/hash-MD5
```

Comment:
As you probably saw, we now have optional MD5 checksumming enabled.

Reply:
Yes, working on it right now.